Library/Homebrew/utils/pypi.rb

```diff
@@ -237,23 +237,16 @@ def self.update_python_resources!(formula, version: nil, package_name: nil, extr
                                     exclude_packages: nil, dependencies: nil, install_dependencies: false,
                                     print_only: false, silent: false, verbose: false,
                                     ignore_errors: false, ignore_non_pypi_packages: false)
-    auto_update_list = formula.tap&.pypi_formula_mappings
-    if auto_update_list.present? && auto_update_list.key?(formula.full_name) &&
-       package_name.blank? && extra_packages.blank? && exclude_packages.blank?
-
-      list_entry = auto_update_list[formula.full_name]
-      case list_entry
-      when false
-        unless print_only
-          odie "The resources for \"#{formula.name}\" need special attention. Please update them manually."
-        end
-      when String
-        package_name = list_entry
-      when Hash
-        package_name = list_entry["package_name"]
-        extra_packages = list_entry["extra_packages"]
-        exclude_packages = list_entry["exclude_packages"]
-        dependencies = list_entry["dependencies"]
```

Comment:
Just confirming that this won't break anything in the interim: homebred-core still has a `pypi_formula_mappings.json` in its repo root, so do we need to temporarily fall back to that until we're done copying the mappings into each formula? Or is this fine because homebred-core shouldn't be living on the `HEAD` of brew anyways?